Foxtrot/Foxtrot/Extraction/ExtractorVisitor.cs

```diff
@@ -1453,6 +1453,33 @@ public TypeNode ClosureClass
                 }
             }
 
+            public override void VisitReturn(Return Return)
```

Comment:
I think better argument name would be `@return` or `returnStatement`, because `Return` violates common .NET naming guidelines...